<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QMainWindow" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="dashboardCentralWidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Weclome, &lt;username&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="clockLabel">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="totalNotesGroupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Total Notes</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="recentNotesGroupBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Recent Notes</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="reportsGroupBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Reports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notesButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>361</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newNoteButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>490</y>
      <width>361</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>New Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportsButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>490</y>
      <width>361</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reports</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="dashboardMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1196</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>â€Ž </string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="dashboardStatusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QMainWindow" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="dashboardCentralWidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Weclome, &lt;username&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="clockLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="totalNotesGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Total Notes</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="recentNotesGroupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Recent Notes</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="reportsGroupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Reports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>221</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newNoteButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>New Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportsButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>221</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reports</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="dashboardMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="dashboardStatusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
>>>>>>> f29377b6a49b3dc9fa486f0f187a3e82ccd13fde
